docs/getting-started/new-project-guide/jvm_lang.md

```diff
@@ -0,0 +1,168 @@
+---
+layout: default
+title: Integrating a Java/JVM project
+parent: Setting up a new project
+grand_parent: Getting started
+nav_order: 3
```

Comment:
Python is already 3rd.
Can you change this to 4 and then update https://github.com/google/oss-fuzz/blob/master/docs/getting-started/new-project-guide/bazel.md to be 5th?